ownCloud/Viewer/WebViewDisplayViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ImageDisplayViewController.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 30/08/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+import ownCloudSDK
+import WebKit
+
+class WebViewDisplayViewController: DisplayViewController, DisplayViewProtocol {
+	static var supportedMimeTypes: [String] =
+		["image/jpeg",
+		 "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+		 "application/vnd.openxmlformats-officedocument.wordprocessingml.document"]
+
+	static var features: [String : Any]? = [FeatureKeys.canEdit : true]
+
+	static func webViewExtension(identifier: String) -> OCExtension {
```

Comment:
Seeing how this method and `normalPDFExtension` are almost identical, I can see how this could be unified and simplified, f.ex. with an extension to `DisplayViewProtocol` like this:

```swift
extension DisplayViewProtocol {
	static var displayExtension: OCExtension {
            // Code generating OCExtension from `supportedMIMETypes`, `features` and `displayExtensionIdentifier` goes here.
	}
}
```

Then, adding the extension to the extension manager can be done with one line

```swift
OCExtensionManager.shared.addExtension(PDFViewerViewController.displayExtension)
```

and everything related to the respective viewer (incl. ID) is isolated in its own class.